hub/jupyterhub_config.py

```diff
@@ -6,26 +6,19 @@
 c.JupyterHub.proxy_cmd = '/usr/local/bin/nchp'
 c.JupyterHub.ip = '0.0.0.0'
 
-c.KubeSpawner.kube_namespace = os.environ.get('POD_NAMESPACE', 'default')
-c.KubeSpawner.kube_api_endpoint = 'https://{host}:{port}'.format(
-    host=os.environ['KUBERNETES_SERVICE_HOST'],
-    port=os.environ['KUBERNETES_SERVICE_PORT']
-)
+c.KubeSpawner.namespace = os.environ.get('POD_NAMESPACE', 'default')
+
 # Disable SSL Auth for now. It's not the end of the world because we
 # are inside the cluster, but we should fix this by moving to kubesession
 # soon! FIXME
-c.KubeSpawner.kube_ca_path = False
```

Comment:
Just wondering, the deployment works without this line? That makes sense because I believe GCE will automatically use SSL auth in the cluster but I just wanted to make sure.

If so, we can also delete the comment above.